src/content/learn/state-as-a-snapshot.md

```diff
@@ -61,43 +61,43 @@ label, textarea { margin-bottom: 10px; display: block; }
 
 </Sandpack>
 
-Here's what happens when you click the button:
+Inilah yang terjadi ketika Anda menekan tombol "Kirim" pada contoh diatas:
 
-1. The `onSubmit` event handler executes.
-2. `setIsSent(true)` sets `isSent` to `true` and queues a new render.
-3. React re-renders the component according to the new `isSent` value.
+1. *Event handler* `onSubmit` dijalankan.
+2. `setIsSent(true)` mengubah nilai `isSent` menjadi `true` dan memasukkan antrian *render* baru.
+3. React melakukan *render* ulang pada komponen tersebut sesuai dengan nilai `isSent` yang baru.  
 
-Let's take a closer look at the relationship between state and rendering.
+Mari kita lihat lebih dekat hubungan antara *state* dan *rendering*.
 
-## Rendering takes a snapshot in time {/*rendering-takes-a-snapshot-in-time*/}
+## Rendering mengambil sebuah snapshot pada waktu tersebut {/*rendering-takes-a-snapshot-in-time*/}
```

Comment:
Ini enaknya apa ya? `Snapshot in time` @mhaidarhanif @zainfathoni @r17x @resir014 ?

Reply:
```suggestion
## Rendering mengambil sebuah snapshot pada waktu render tersebut {/*rendering-takes-a-snapshot-in-time*/}
```

Based on the translation a few lines after this, we can do it like this.